### DISCUSSION TITLE
Change Default Image

### DISCUSSION THREAD

**Discussion by 1CharlesStern** [farling42/fvtt-pdf-pager#110] [Ideas]
Would you be interested in a feature contribution for PDF Pager?

I'd like to put in a UI element to change the default image of an actor.  It's something I've had issues with, mostly on PDFs that don't support a space for a default image.  I'd probably put it in the header next to 'Log PDF fields.'

**Comment by farling42** [comment]
I can certainly have a look at the contribution.

**Comment by farling42** [comment]
Version 1.3.0 adds an additional button to the Actor/Item sheet's title bar to allow the Portrait to be modified.
